lib/helper/TokenHolder.js

```diff
@@ -79,6 +79,21 @@ class TokenHolder {
     return executableData;
   }
 
+  /**
+   * Logout all authorized sessions.
+   *
+   * @returns {*} Executable data to logout sessions.
+   */
+  getLogoutExecutableData() {
+    const oThis = this;
+
+    const jsonInterface = oThis.abiBinProvider.getABI(tokenHolderContractName),
+      contract = new oThis.auxiliaryWeb3.eth.Contract(jsonInterface, oThis.tokenHolderProxy),
+      executableData = contract.methods.logout().encodeABI();
```

Comment:
Let's refactor below two lines in a single method and call it from different methods in this file:
```
const jsonInterface = oThis.abiBinProvider.getABI(tokenHolderContractName),
      contract = new oThis.auxiliaryWeb3.eth.Contract(jsonInterface, oThis.tokenHolderProxy),
```
It's used in multiple methods.

Reply:
`getTokenHolderInstance` method added.